sdk/nodejs/tests/provider/experimental/testdata/bad-property-type/index.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright 2025-2025, Pulumi Corporation.  All rights reserved.
+
+import * as pulumi from "@pulumi/pulumi";
+
+export interface MyComponentArgs {
+    invalidProp: pulumi.Input<boolean | string>;
+}
```

Comment:
Another test in addition to the map one above that would be nice to have would be 
```suggestion
    complexProp: MyOtherArgs;
}

export interface MyOtherArgs {
    invalidProp: pulumi.Input<boolean | string>;
}
```

Reply:
Great idea, added